noxfile.py

```diff
@@ -9,6 +9,7 @@
 # ignored.
 TYPED_FILES = {
     "src/urllib3/contrib/__init__.py",
+    "src/urllib3/contrib/_securetransport/low_level.py",
```

Comment:
Forgot to mention it in the previous PR, can we add the `bindings.py` to this list? Just so we run mypy on it even if it's basically skipped.